### PR TITLE
:construction_worker: Added Codecov to the build.yml

### PR DESCRIPTION
**Pull request by sunnyosun**

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=73&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=73&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=73&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty)

#### Legend
CodeSee Map Legend

**Comment by falexwolf** [comment]
Thank you, Sunny! ❤️

File: noxfile.py [hunk 1]
```diff
@@ -23,3 +23,7 @@ def build(session):
     prefix = "." if Path("./lndocs").exists() else ".."
     session.install(f"{prefix}/lndocs")
     session.run("lndocs")
+    session.install("coverage")
+    session.run("coverage", "run", "-m", "pytest", "tests")
```
Comment (falexwolf) [inline noxfile.py:27]: Hey! 👋 ❤️ 

This whole PR is awesome and the solution seems very elegant already.

I just don't fully understand why coverage doesn't need to know about `--nbmake` in the `pytest` call above.

Does it 
1. re-execute the tests to compute coverage without nbmake and that's why coverage is pretty low?
2. simply look into the test statistics that pytest had written before, and the 54% test statistics come from executing the notebooks?

Comment (falexwolf) [reply noxfile.py:27]: In any case: I'd hope that tests through notebooks/guides/tutorials add to the coverage that results from standard tests in the `tests` directory.

**Pull request merged**